molecule/docker/create.yml

```diff
@@ -0,0 +1,82 @@
+- name: Create
```

Comment:
Was this docker stuff created before `molecule_docker_ci` existed?

Reply:
Hmm, I'm going to go with "yes". Ultimately, there's not going to be any Docker testing of this role and this lives only for lint purposes. I still haven't figured out if OpenStack will be sufficient for this role to test or not.

Reply:
Assuming openstack is sufficient, I don't think there's a need to keep around the docker scenario just for lint. The openstack scenario should also lint just fine, unless you want an explicit "lint only" scenario that doesn't have requirements or something.

Reply:
I've about figured out how to make this happen with OpenStack. So expect that in the next set of updates.